capability.go

```diff
@@ -0,0 +1,29 @@
+package omise
+
+// Capability represents Omise's capability object.
+// See https://www.omise.co/capability-api for more information.
+type Capability struct {
+	Object                   string          `json:"object"`
+	Location                 *string         `json:"location"`
```

Comment:
why use pointer type on this field?

Reply:
@choestelus This field is optional, so we use nillable types.

Reply:
@choestelus I just checked the API doc again. This field is not optional, so we can remove it.